pkg/deployment/agency/definitions.go

```diff
@@ -26,4 +26,6 @@ const (
 	ArangoKey          = "arango"
 	PlanKey            = "Plan"
 	PlanCollectionsKey = "Collections"
+	PlanSupervision    = "Supervision"
+	PlanMaintenance    = "Maintenance2"
```

Comment:
This looks odd, should it be `"Maintenance"` or is it needed at all?

Reply:
Obsolete, will cleanup it